src/main/kotlin/io/zerobase/smarttracing/resources/DevicesResource.kt

```diff
@@ -10,9 +10,9 @@ import javax.ws.rs.core.MediaType
 data class CreateDeviceRequest(@JsonProperty("fingerprint") val fingerprint: Fingerprint)
```

Comment:
This requires me to send {"fingerprint": {"value": fingerprint}}, i suggest for requests to just have them as simple strings and convert to the type we use for ourselves later

Reply:
Is this because of the @JsonProperty annotation? I have removed that now.

Reply:
yeah, i forgot to mention it, but the inline classes aren't handled by jackson right now. in theory, it should just unwrap it and stick the raw value in there, but it doesn't. that's why all the resource methods take strings and not the id wrappers.

if the wrappers are too much of a pain, we can always drop them and just use strings.